nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -45,6 +46,7 @@ WaypointFollower::on_configure(const rclcpp_lifecycle::State & /*state*/)
 
   stop_on_failure_ = get_parameter("stop_on_failure").as_bool();
   loop_rate_ = get_parameter("loop_rate").as_int();
+  sleep_time_inbetween_waypoints_ = get_parameter("sleep_time_inbetween_waypoints").as_int();
```

Comment:
This should be a double, not an int

Reply:
I also thought of double at first, but using double with `time` is troublesome and often will need a casting anyway, I suggest we stick with int, though we can change the  unit to milliseconds ?

Reply:
I don’t think it matters. Up to you. Int is OK for ms, but double for seconds.